source/ui/win32/Win32DisplayWindow.ooc

```diff
@@ -18,7 +18,7 @@
 import DisplayWindow
 use ooc-math
 use ooc-draw
-use ooc-ui
+import Win32Window
 
 version(windows) {
 Win32DisplayWindow: class extends DisplayWindow {
```

Comment:
And this one has `DisplayWindow` while the Unix equivalent has no `Display` in its name. But no, it shouldn't be fixed here and now.